### ISSUE TITLE
Button states

### ISSUE THREAD

**Issue by zcjess** [zcjess/GradeCraft#1]
Should they be tabs?
announce pressed buttons, aria-pressed, to let users know which page they are currently on.

<img width="313" alt="image" src="https://github.com/zcjess/GradeCraft/assets/90011962/a6127451-010e-4b5e-a329-6e359879aa32">
https://staging.gradecraft.com/courses/1144/dashboard

<img width="456" alt="image" src="https://github.com/zcjess/GradeCraft/assets/90011962/1b29fa22-6c7f-4eec-a829-57c9f8ed5010">
https://staging.gradecraft.com/overview
<img width="821" alt="image" src="https://github.com/zcjess/GradeCraft/assets/90011962/96aca0fe-2737-40e7-af38-04c092a0773e">

**Comment by zcjess** [comment]
1. Should have aria-haspopup and aria-expanded labels
2. Should be able to close the popup by clicking the button again, right now no way to close it
<img width="1274" alt="image" src="https://github.com/zcjess/GradeCraft/assets/90011962/50b70e51-9ab6-47f5-81f0-944369f29568">

**Comment by zcjess** [comment]
Should it have aria-expanded?
<img width="132" alt="image" src="https://github.com/zcjess/GradeCraft/assets/90011962/dcf58e64-6c9d-4024-8323-6dd07427d5a2">
https://staging.gradecraft.com/courses/1144/predictor

**Comment by zcjess** [comment]
No aria labels for expandable buttons and status. The button can't be expanded using space.
![image](https://github.com/zcjess/GradeCraft/assets/90011962/bfc05523-c5f2-4296-b379-7eed8f66dd01)

**Comment by zcjess** [comment]
use aria labels with pop sound
should move focus into the pop up
![image](https://github.com/zcjess/GradeCraft/assets/90011962/25c286ac-3ada-40c4-bff5-e90372fc53dc)
![image](https://github.com/zcjess/GradeCraft/assets/90011962/0e6003d3-14f6-4d2a-8429-148a5aae9190)

**Comment by zcjess** [comment]
on/off status
![image](https://github.com/zcjess/GradeCraft/assets/90011962/76d46117-e93f-49ff-a967-8d93d99ca8c4)

**Comment by zcjess** [comment]
ideally announce the new number when users click on "+" or "-"
![image](https://github.com/zcjess/GradeCraft/assets/90011962/16ba148e-1ba3-4ff4-aced-8e56ef87dc6f)
